core/domain/exp_jobs_one_off.py

```diff
@@ -312,10 +312,12 @@ def map(item):
             if state.interaction.id == 'ItemSelectionInput':
                 choices = (
                     state.interaction.customization_args['choices']['value'])
+                html_list = [translatable_html['html'] for (
```

Comment:
Unfortunately "translatable_html" still sounds like a string.

Maybe html_with_content_id or annotated_html perhaps?

Reply:
we have subtitled HTML, will it be okay to rename this to same?

Reply:
Does subtitled_html have exactly the same internal structure as translatable_html? If so, then yes.